src/memfs.ts

```diff
@@ -129,7 +130,8 @@ export class InMemoryFileSystem extends EventEmitter implements ts.ParseConfigHo
 	 * @param path File path or URI (both absolute or relative file paths are accepted)
 	 */
 	fileExists(path: string): boolean {
-		return this.readFileIfExists(path) !== undefined || this.files.has(path) || this.files.has(path2uri(this.path, path));
+		const uri = path2uri(path);
+		return this.files.has(uri) || typeScriptLibraries.has(path);
```

Comment:
yeah, but in this place it was already taken into account because `readFileIfExists` does
Don't know about the other place